packages/web-components/src/components/va-additional-info/va-additional-info.tsx

```diff
@@ -37,6 +41,11 @@ export class VaAdditionalInfo {
   })
   componentLibraryAnalytics: EventEmitter;
 
+  @Listen('resize', { target: 'window' })
+  handleResize() {
+    this.updateInfoMaxHeight();
+  }
+
```

Comment:
This is required to make sure max height is correct if user resizes/zooms in/out.